CMakeLists.txt

```diff
@@ -86,12 +108,11 @@ if(${HAVE_MYSOFA})
 endif(${HAVE_MYSOFA})
 
 configure_file(
-  "${CMAKE_CURRENT_SOURCE_DIR}/cmake/spatialaudio.pc.cmake"
-  "${CMAKE_CURRENT_BINARY_DIR}/${PROJECT_NAME}.pc"
-  @ONLY
+    "${CMAKE_CURRENT_SOURCE_DIR}/cmake/spatialaudio.pc.cmake"
+    "${CMAKE_CURRENT_BINARY_DIR}/${PROJECT_NAME}.pc"
+    @ONLY
```

Comment:
This should also be in a separated commit

Reply:
Also wouldn't it make sense to ship 2 .pc files now? (To be fair I'm not sure what the recommended/idiomatic way to do this is)